src/cljs/rems/table.cljs

```diff
@@ -123,29 +125,44 @@
                               (s/optional-key :sort-order) (s/enum :asc :desc)
                               (s/optional-key :set-sorting) Applicable}
    (s/optional-key :filtering) {(s/optional-key :filters) s/Str
+                                (s/optional-key :filters-new) s/Str
                                 (s/optional-key :show-filters) s/Bool
                                 (s/optional-key :set-filtering) Applicable}
    :id-function Applicable
    :items [s/Any]
    (s/optional-key :class) s/Str})
 
 (defn- filter-view [filtering]
-  (let [{:keys [filters set-filtering]} filtering]
+  (let [{:keys [filters-new set-filtering]} filtering
+        update-current-filters (fn [event]
+                                 (set-filtering (assoc filtering :filters filters-new)))]
+    [:div.flex-grow-1.d-flex
```

Comment:
The indentation doesn't match the parentheses in multiple places in this file and thus breaks when the file is opened with parinfer. To a code reformat for the whole file.